index.html

```diff
@@ -14,10 +14,10 @@
 			<h1>Todo List</h1>
 			<time id="todayDate"></time>
 		</div>
-		<div class="input">
+		<form id="taskForm" onsubmit="addTask(event)">
```

Comment:
This is the old html/js.

This it's the best way to do it, you should search a little about eventListerns: https://www.w3schools.com/js/js_htmldom_eventlistener.asp

```suggestion
		<form id="taskForm">
```

Reply:
`submit` it's a event

Reply:
okok